lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -137,6 +137,16 @@ CRI_F2F:
       targetState: F2F_HANDOFF_PAGE
     access-denied:
       targetState: PYI_ANOTHER_WAY
+    not-found:
+      targetState: ERROR
+    temporarily-unavailable:
+      targetState: ERROR
+    fail-with-no-ci:
+      targetState: ERROR
+    end:
+      targetState: ERROR
+    enhanced-verification:
+      targetState: ERROR
```

Comment:
Interestingly, once we add F2F CIMIT this should become `F2F_HANDOFF_PAGE` (because the F2F is a valid enhanced verification journey).